app/js/handlebarsScoreTable.ts

```diff
@@ -322,9 +322,14 @@ async function addEventListenersForViewResults() {
             addEventListenersForCloseResults();
             getData("result?id=" + e.target.getAttribute("dataId")).then(resultData => {
                 getData("user").then(userData => {
-                    userData.data.forEach(user =>{
+                    userData.data.forEach(user => {
                         if (user.id === resultData.data.id) {
                             let testId = user.test_id
+                            if (testId !== '1') {
+                                hideButton()
+                            } else if (testId == '1') {
```

Comment:
you dont need the if here, it must be `'1'` if the above if is false. infact, you don't need the else at all. a better solution would be to hide the buttons in the CSS by default. then only show them if the test id is 1. this will make a smoother UX and give you less JS